freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/PasswordService.java

```diff
@@ -63,7 +63,7 @@ public class PasswordService {
     private GrpcUmsClient umsClient;
 
     @Inject
-    @Qualifier(UsersyncConfig.USERSYNC_TASK_EXECUTOR)
+    @Qualifier(UsersyncConfig.USERSYNC_EXTERNAL_TASK_EXECUTOR)
     private ExecutorService asyncTaskExecutor;
```

Comment:
I think it would be more clear if these were renamed based on the `@Qualifier`. For the other occurrences as well.